index.html

```diff
@@ -231,6 +231,9 @@ <h3>
       the [=digital credential/request data=] to be handled by the holder's
       software, such as a digital wallet.
     </p>
+    <p>
+      The object MUST only contain [=JSON type=] values.
+    </p>
```

Comment:
We probably don't need this. Serialization will throw automatically if it's not the case. 

```suggestion
```

Reply:
Yes, but it's in the algorithm. It's redundant to state it twice.

Reply:
Where it should be defined is part of the registry rules. That is, for your protocol to be registered, it might be JSON stringifiable. That would let new protocols know that it's a requirement.